jobs/build_docker/get_docker_commit_version.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash 
+set -e
+
+build_record=`ls ${DOCKER_RECORD_PATH}`
+image_list=`head -n 1 $build_record`
+docker_repo_commit_file="${DOCKER_REPO_HASHCODE_FILE}"
```

Comment:
why don't you pass in "${DOCKER_REPO_HASHCODE_FILE}" as $1 ?
using ENV Variable to pass variable from shell script to shell script is a bad idea. which is hard to tell what parameters of a script is needed as a input param.